<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>p_pwindow</class>
 <widget class="QMainWindow" name="p_pwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>780</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>六子棋</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_choice">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_restart"/>
    <addaction name="action_return"/>
   </widget>
   <widget class="QMenu" name="menu_hint">
    <property name="title">
     <string>提示</string>
    </property>
    <addaction name="action_dohint"/>
    <addaction name="action_undohint"/>
   </widget>
   <addaction name="menu_choice"/>
   <addaction name="menu_hint"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_undo">
   <property name="text">
    <string>悔棋</string>
   </property>
  </action>
  <action name="action_stale">
   <property name="text">
    <string>求和</string>
   </property>
  </action>
  <action name="action_restart">
   <property name="text">
    <string>重新开始</string>
   </property>
  </action>
  <action name="action_dohint">
   <property name="text">
    <string>开启禁手提示</string>
   </property>
  </action>
  <action name="action_undohint">
   <property name="text">
    <string>关闭禁手提示</string>
   </property>
  </action>
  <action name="action_return">
   <property name="text">
    <string>返回菜单</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
